.github/workflows/triage.yml

```diff
@@ -162,7 +162,7 @@ jobs:
               keep_if_no_match: true
 
             - label: CI-linux-self-hosted
-              path: Formula/(gdbm|go|openssl@1.1|python@3.9).rb
+              path: Formula/(gdbm|go|openssl@1.1|python@3.10).rb
```

Comment:
Can you create a separate PR for this? Previous PR runs show that `python@3.10` needs `CI-linux-self-hosted`. Last `python@3.9` run (#107088) took less than 2 hours, so most likely doesn't need self-hosted anymore.